sos/report/plugins/maas.py

```diff
@@ -67,58 +62,59 @@ def _is_snap_installed(self):
             return maas_pkg['pkg_manager'] == 'snap'
         return False
 
+    def _snap_collect(self):
+        self.add_cmd_output([
+            'snap info maas',
+            'maas status'
+        ])
+
+        self.add_forbidden_path([
+            "/var/snap/maas/**/*.key",
+            "/var/snap/maas/**/*.pem",
+            "/var/snap/maas/**/secret",
+        ])
+
+        self.add_copy_spec([
+            "/var/snap/maas/common/log",
```

Comment:
with the improvements we've been discussing, is it worth splitting this apart so that we collect recent logs normally, and we collect all when we specify `--all-logs`

Reply:
Updated to keep only latest `*.log` (not to include `*.log.1` and others)
Added logic to collect only recent syslog data for each machine (if `--all-logs` not provided)